content/the-ember-times-issue-178.md

```diff
@@ -26,15 +28,14 @@ tags:
 
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## Addon roundup 🙌
```

Comment:
could also change to `Addon hodgepodge roundup`, if not too weird 😅

Reply:
I think Addon roundup sounds fine. 🙂